feature-libs/product/configuration/src/textfield-configurator/facade/configurator-textfield.service.ts

```diff
@@ -33,13 +33,15 @@ export class ConfiguratorTextfieldService {
     owner: GenericConfigurator.Owner
   ): Observable<ConfiguratorTextfield.Configuration> {
     this.store.dispatch(
-      new ConfiguratorActions.CreateConfiguration({
+      new ConfiguratorTextfieldActions.CreateConfiguration({
         productCode: owner.id, //owner Id is the product code in this case
         owner: owner,
       })
     );
 
-    return this.store.select(ConfiguratorSelectors.getConfigurationContent);
+    return this.store.select(
```

Comment:
Let's use `this.store.pipe(select(...))` instead. I believe all pipeable operators are tree-shakeable